web/client/localConfig.json

```diff
@@ -50,6 +50,11 @@
                 "type": "csw",
                 "title": "Demo CSW Service",
                 "autoload": true
+              },
+                "Map Backgrounds": {
```

Comment:
This background's catalog(s) should be added to the selectable catalogs list when your're adding backgrounds.

Here you're configuring it (for default catalog), assuming it is named "Map Backgrouds" and you remove it for normal use. 
This wrong approch causes many issues: 
- this catalog(s) will appear only on new maps, or maps saved with this catalog inside (so existing maps can not see this catalog)